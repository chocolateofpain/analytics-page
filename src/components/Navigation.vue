<template>
  <div class="navigation">
    <div class="nav-link-empty"/>
    <Typography as="small" class="nav-link-reporting" :uppercase="true">
      Reporting
    </Typography>
  </div>
</template>

<script>
import Typography from './library/typography/Typography.vue'

export default {
  name: 'Navigation',
  components: {
    Typography
  }, 
  props: [
    'checksCategory'
  ]
}
</script>

<style scoped>
.navigation {
  background-color: #E8E8E8;
  margin-right: 69px;
  padding: 16px;
  width: 174px;
}

.nav-link-reporting {
  background-color: #C4C4C4;
  border-radius: 4px;
  display: block;
  padding-left: 14px;
  padding-top: 9px;
  height: 35px;
  width: 140px;
}

/* dummy element to mock navigation */
.nav-link-empty {
  background-color: #C4C4C4;
  border-radius: 4px;
  margin-bottom: 20px;
  height: 35px;
  width: 60px;
}
</style>
