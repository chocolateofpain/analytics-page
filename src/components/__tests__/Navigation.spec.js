import { mount } from '@vue/test-utils'
import Navigation from '../Navigation.vue'

describe('Navigation', () => {
  it('should be defined', () => {
    expect(Navigation).toBeDefined()
  })

  describe('snapshots', () => {
    it('should render component', () => {
      const component = mount(Navigation)
      expect(component).toMatchSnapshot()
    })
  })
})
