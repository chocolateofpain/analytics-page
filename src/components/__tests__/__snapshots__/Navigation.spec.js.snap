// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation snapshots should render component 1`] = `
<div class="navigation">
  <div class="nav-link-empty"></div> <small class="nav-link-reporting typography typography--small   typography--uppercase   ">
    Reporting
  </small>
</div>
`;
